src/doc/guide.md

```diff
@@ -4231,7 +4231,7 @@ arguments, really powerful things are possible.
 
 Let's make a closure:
 
-```{rust}
+```{rust,ignore}
```

Comment:
@japaric would you prefer a PR from me to your PR, or do you just want to change the syntax and I can land the language after?